build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -34,6 +34,20 @@ data:
     cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
     mkdir -force c:/k/antrea/bin
     cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+  Install-OVSDriver-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    mkdir -force c:/openvswitch/driver
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+    $OVSDriverDir = "$mountPath\openvswitch\driver"
+
+    # Check if OVSExt driver is already installed
+    $driverStatus = netcfg -q ovsext
+    if ($driverStatus -like '*not installed*') {
+      # Install OVS Driver
+      netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
```

Comment:
OVS driver installation must succeed to run ovs processes, would prefer to add try catch in .ps1, to capture some debugging message if driver failed to install or succeeded
try {
 netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
Write-Output "OVSExt driver has been installed successfully."
    } catch {
        Write-Error "Failed to install OVSExt driver: $_"
    }

Reply:
> OVS driver installation must succeed to run ovs processes, would prefer to add try catch in .ps1, to capture some debugging message if driver failed to install or succeeded try { netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt Write-Output "OVSExt driver has been installed successfully." } catch { Write-Error "Failed to install OVSExt driver: $_" }

@rajnkamr I tested and found that netcfg does not generate a terminating error upon failure, so `try-catch` didn’t catch errors. Therefore, I checked whether the command was successful by searching for failure content in the command result.